internal/apis/certmanager/types_issuer.go

```diff
@@ -232,6 +238,28 @@ type VaultAppRole struct {
 	SecretRef cmmeta.SecretKeySelector
 }
 
+// VaultKubernetesAuth is used to authenticate against Vault using a client
+// certificate stored in a Secret.
+type VaultClientCertificateAuth struct {
+	// The Vault mountPath here is the mount path to use when authenticating with
+	// Vault. For example, setting a value to `/v1/auth/foo`, will use the path
+	// `/v1/auth/foo/login` to authenticate with Vault. If unspecified, the
+	// default value "/v1/auth/cert" will be used.
+	// +optional
```

Comment:
I noticed that `VaultAppRole` and `VaultKubernetesAuth` do not have the `+optional` hint for `Path`, even though they also have a default value, if it is unset.
Should I remove the `+optional` from here or should it also be added to these two?